UnitTests/BraintreeCoreTests/BTAnalyticsService_Tests.swift

```diff
@@ -218,14 +218,14 @@ final class BTAnalyticsService_Tests: XCTestCase {
             XCTAssertEqual(mockAnalyticsHTTP.POSTRequestCount, 1)
 
             let parameters = mockAnalyticsHTTP.lastRequestParameters?["analytics"] as? [[String: Any]]
-            let timestampOne = parameters?[0]["timestamp"] as! Int
-            let timestampTwo = parameters?[1]["timestamp"] as! Int
+            let timestampOne = Int(parameters?[0]["timestamp"] as! Double)
+            let timestampTwo = Int(parameters?[1]["timestamp"] as! Double)
```

Comment:
We can revert the need to double cast here, once we update the source to use Int (UInt64) instead of Double.

Reply:
Was already updated: https://github.com/braintree/braintree_ios/pull/887/commits/bc9cead62a7f3a9bca5bb74387f0501091c9971c